### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by maqnouch**

### CODE REVIEW

File: README.textile [hunk 1]
```diff
@@ -7,13 +7,13 @@ h3. "https://www.elastic.co/products/elasticsearch":https://www.elastic.co/produ
 Elasticsearch is a distributed RESTful search engine built for the cloud. Features include:
 
 * Distributed and Highly Available Search Engine.
-** Each index is fully sharded with a configurable number of shards.
```
Comment (clintongormley) [inline README.textile]: sharded is correct here

File: README.textile [hunk 2]
```diff
@@ -7,13 +7,13 @@ h3. "https://www.elastic.co/products/elasticsearch":https://www.elastic.co/produ
...
+** Each index is fully shared with a configurable number of shards.
 ** Each shard can have one or more replicas.
 ** Read / Search operations performed on either one of the replica shard.
 * Multi Tenant with Multi Types.
 ** Support for more than one index.
 ** Support for more than one type per index.
-** Index level configuration (number of shards, index storage, ...).
+** Index level configuration (number of shards, index storage …).
```
Comment (clintongormley) [inline README.textile]: I'd prefer to keep the README as ASCII only

**Comment by clintongormley** [comment]
Hi @maqnouch 

Thanks for the PR. Please note my comments above, and push a fixed version? And please could I ask you to sign the CLA?
http://www.elasticsearch.org/contributor-agreement/

**Comment by maqnouch** [comment]
done thank you for your reply.

**Comment by clintongormley** [comment]
@maqnouch please push a new commit once you have addressed my comments above.

**Comment by clintongormley** [comment]
no update here. closing

**Pull request closed without merging**